### PR TITLE
Additional email templates added

### PR DESCRIPTION
**Pull request by jesseyuen**
Adding the remaining 4 email templates to this suite of notifications! 🎉

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 1]
```diff
@@ -85,10 +85,112 @@ The Becoming a Teacher team
...
+## Email template: A course has been created
+
+{% set template6 %}
+Dear colleague,
+
+A new course has been created in Publish teacher training courses, and your organisation has been added as the accredited body for it.
+
+((Provider_name)) created ((course_name)) ((course_code)) at ((create_course_datetime)). The course has not yet been published on Find postgraduate teacher training. You’ll receive a notification if it is published.
+
+View the course at ((publish_url)).
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md:101]: Where will this link to?

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md [hunk 2]
```diff
@@ -85,10 +85,112 @@ The Becoming a Teacher team
...
+
+Your organisation has been added as the accredited body for a course in Find postgraduate teacher training.
+
+The accredited body for ((course_name)) (((course_code))) was previously ((old accredited body)). This was changed in Publish teacher training courses by ((provider_name)) at ((attribute_change_datetime)).
+
+View the course at ((course_url)).
+
+If you have any questions about this, please notify your contact at ((provider_name)).  
+
+To stop receiving these notifications, or to update your settings, contact becomingateacher@digital.education.gov.uk, or reply to this email.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]: Can't they update their settings on Publish?

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md (hunk 1 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md:101]: It's a placeholder variable(?), it goes to the courses as an accredited body screen.

File: app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md (hunk 2 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md]: Yes, the templates are actually being updated now to refelct this. I added the originally designed templates. I'lll update this!

**Review by fofr — APPROVED** [review]

**Pull request merged**